scripts/ps/check_golint.ps1

```diff
@@ -0,0 +1,7 @@
+# check_golint.ps1
+# Run the linter on everything
+
+$out = & golint -set_exit_status $(& go list ./...)
+if ($LastExitCode -ne 0) {
+    Write-Error $out
+}
```

Comment:
one more new-line please, you can use inline code editing for that.
it looks awesome @marpio